lib/livebook_web/live/output.ex

```diff
@@ -243,19 +243,25 @@ defmodule LivebookWeb.Output do
     )
   end
 
-  defp render_output({:error, formatted}, %{}) do
-    assigns = %{message: formatted}
+  defp render_output({:error, formatted, :unavailable_secret}, %{}) do
+    assigns = %{message: String.replace(formatted, ~s(environment variable "LB_), ~s(secret "))}
```

Comment:
I think we shouldn't rewrite the message/stacktrace, especially that in the code we actually try to read an env variable with `LB_` prefix.

Reply:
I asked @cristineguadelupe to rewrite the message because we are trying to avoid leaking the implementation detail as much as possible, let’s see how it works out :)

Reply:
Hmm, I don't like that someone writing `System.fetch_env!("LV_PASSWORD")` would get a different error from `System.fetch_env!("PASSWORD")`. Also, matching on the specific error in livebook is much more coupling between elixir runtime and livebook than we generally want.

Reply:
We match in the error message inside the evaluator, which should be Elixir specific, no?

Reply:
By matching I mean that we rewrite this specific error message in this LV.

Reply:
Oh, we should handle this within the runtime then, the same place that sets `unavailable_env`.

Reply:
Currently the runtime doesn't know about secrets, but that's better anyway. I would still keep the error untouched and rather add a separate message mentioning the secret name together with the button (or include the name in the button), your call :)

Reply:
could not fetch the secret `foo` ---- > Add secret 
original error here...

🤔